cpp/Makefile.inc

```diff
@@ -20,9 +20,9 @@ else
 endif
 
 ifeq ($(CCC),clang++)
-  CCCFLAGS += -std=gnu++2a -Wno-vla-extension #-Wno-c++1y-extensions
+  CCCFLAGS += -std=gnu++20 -Wno-vla-extension #-Wno-c++1y-extensions
```

Comment:
This has already changed in `ubuntu2204` branch. Please execute `git pull origin ubuntu2204` and solve all merge conflicts before i can continue the review.